google-business-profie/config.json

```diff
@@ -0,0 +1,261 @@
+{
+    "name": "Google Business Profile",
+    "version": "1.0.0",
+    "description": "Manage Google Business Profile reviews using official Google API - read reviews and reply to customers",
+    "entry_point": "reviews.py",
+    "auth": {
+        "type": "platform",
+        "provider": "Google Maps Reviews",
+        "scopes": [
+            "https://www.googleapis.com/auth/business.manage"
```

Comment:
It seems like no verification is required

Reply:
Yeah that's what I mean - 
<img width="521" height="256" alt="image" src="https://github.com/user-attachments/assets/e86a1be5-7823-4fe7-83f0-62b89cd3bd2f" />
<img width="627" height="355" alt="image" src="https://github.com/user-attachments/assets/cb72ab61-c794-4b74-923f-bcc48a053ef1" />